### PR TITLE
Run ispell on docs.

### PR DESCRIPTION
**Pull request by perweij**
I ran ispell on README.org. It found some minor errors. It also proposed a few changes in casing - I can exclude the latter from the PR if you like.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -315,7 +315,7 @@ Treemacs offers the following configuration options (~describe-variable~ will us
...
 | treemacs-indentation-string         | " "                                         | The string that is used to create indentation.                                                                                                                                             |
 | treemacs-width                      | 35                                          | Width of the treemacs window.                                                                                                                                                              |
-| treemacs-show-hidden-files          | t                                           | Dotfiles will be shown if this is set to t and be hidden otherwise.                                                                                                                        |
+| treemacs-show-hidden-files          | t                                           | Dot files will be shown if this is set to t and be hidden otherwise.                                                                                                                        |
```
Comment (Alexander-Miller) [inline README.org]: Changes are all fine, except for this one. Dotfiles should be a single word.

**Comment by perweij** [comment]
Dotfiles is now spelled correctly.

**Comment by Alexander-Miller** [comment]
All right. Squash your commits and I'll merge this.

**Comment by Alexander-Miller** [comment]
bump?

**Comment by Alexander-Miller** [comment]
I did the squashing on my own. The changes are on master now, so I'll close here.

**Pull request closed without merging**